docs/details/commands/pmm-admin.md

```diff
@@ -325,10 +325,28 @@ When you remove a service, collected data remains on PMM Server for the specifie
 
 ##### Advanced Options
 
-###### Enable all collectors
-
 PMM starts the MongoDB exporter by default only with `diagnosticdata` and `replicasetstatus` collectors enabled.
 
+    FLAGS:
+
+    `--enable-all-collectors`
+    :  Enable all collectors
+
+    `--disable-collectors`
+    :  Comma-separated list of collector names to exclude from exporter
+
+    `--max-collections-limit=-1`
+    :  Disable collstats & indexstats if there are more than <n> collections. 0: No limit, -1: Let PMM automatically set this value.
+
+        !!! caution ""
+            To high limit could affect CPU and Memory usage.
```

Comment:
I am unable to understand this sentence:

"To high limit could affect CPU and Memory usage."

Reply:
just a suggestion: 
High value for collection limit could impact CPU and Memory Usage on the client node, try using with `--stats-collections` to limit the scope of collections and DB's metrics to be fetched.